test/integration/wallet-web.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function getAddress() {
+  return 'rQDhz2ZNXmhxzCYwxU6qAbdxsHA4HV45Y2';
+}
+
+function getSecret() {
+  return 'shK6YXzwYfnFVn3YZSaMh5zuAddKx';
```

Comment:
That one is from `integration-test.js` that you've put there. Anyway we don't test against live ripple network anymore, so I was thinking about completely removing `wallet.js`, but left it for local testing.